modules/spo-binding-workloads.adoc

```diff
@@ -47,12 +47,31 @@ spec:
 $ oc label ns my-namespace spo.x-k8s.io/enable-binding=true
 ----
 
-. Delete and re-create the pod to use the `ProfileBinding` object:
+. Define a pod named `test-pod.yaml`:
 +
-[source,terminal,subs="attributes+"]
+[source,yaml]
 ----
-$ oc delete pods test-pod && oc create -f pod01.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+spec:
+  containers:
+  - name: test-container
+    image: quay.io/security-profiles-operator/test-nginx-unprivileged:1.21
+----
+
+. Create the pod:
++
+[source,terminal]
 ----
+$ oc create -f test-pod
```

Comment:
Here we need to define the yaml filename while creating pod which is 'test-pod.yaml'
ex: oc create -f test-pod.yaml